docs/configuration/http_conn_man/route_config/rate_limits.rst

```diff
@@ -25,14 +24,10 @@ stage
 kill_switch_key
   *(optional, string)* The key to be set in runtime to disable this rate limit configuration.
```

Comment:
This might not pertain to this PR, by anyway.
the term "kill switch" seems to imply some mechanism to shutoff a system completely before it damages something. My first impression upon reading this is, something like fencing the service instance completely.

However, what you mean here is to kill off the system that limits the flow of requests and let requests flow unhindered. [I hope you see where I am going with this. The term and implication dont align well]. If at all possible, can this field be renamed to "disable_flag" or something?